### PR TITLE
Chapter 6 

### PR DESCRIPTION
**Pull request by linhsasukeno1**
Chapter_6 complete modeling-user
content:
validate user password email
save password by bcrypt in database

### CODE REVIEW

**Comment by linhsasukeno1** [comment]
Ready

File: app/helpers/static_pages_helper.rb [hunk 1]
```diff
@@ -0,0 +1,2 @@
+module StaticPagesHelper
+end
```
Comment (vanvtt-0952) [inline app/helpers/static_pages_helper.rb:2]: file này xóa

File: test/models/user_test.rb [hunk 2]
```diff
@@ -0,0 +1,7 @@
+require 'test_helper'
```
Comment (vanvtt-0952) [inline test/models/user_test.rb:1]: xóa file

**Comment by vanvtt-0952** [comment]
merged
xóa 2 file rồi mợt đi em

**Pull request merged**